packages/plaid-funding-source/pages/index.tsx

```diff
@@ -131,6 +156,20 @@ const Home: NextPage = () => {
         }
     }, [createLinkToken, linkToken]);
 
+    /**
+     * Inital request to create ODA when page loads
+     */
+    useEffect(() => {
+        const onDemandAuthRequuest = async () => {
```

Comment:
Name is misspelled. Should be `onDemandAuthRequest`.

Reply:
Thanks!